app/scripts/directives/serviceInstanceBindings.js

```diff
@@ -9,6 +9,7 @@ angular.module('openshiftConsole').component('serviceInstanceBindings', {
   ],
   controllerAs: '$ctrl',
   bindings: {
+    isOverview: '<?',
```

Comment:
Maybe call this `showParameters` and flip the logic?

Reply:
We can but isOverview is already used in the sub-component  (serviceBinding) so this is really just a pass thru to that.

Reply:
I'm OK with it then